src/main/java/seedu/parser/IndexParser.java

```diff
@@ -0,0 +1,31 @@
+package seedu.parser;
+
+import seedu.contact.Contact;
+import seedu.contact.ContactList;
+import seedu.exception.MissingArgException;
+
+
+public class IndexParser {
+    private static final int COMD_WORD_LENGTH = 1;
+    private static final int INDEX_POSITION = 1;
+    private static final int SIGNIFICANT_INDEX_POSITION = 0;
+    private static final int COMD_INDEX_LENGTH = 2;
+
+
+    public static int getIndexFromInput(String userInput, String command)
+            throws NumberFormatException, MissingArgException {
+        String[] comdIndexSplit = userInput.split(" ", COMD_INDEX_LENGTH);
```

Comment:
Can consider changing the variable name to `destructuredInputs` for consistency with the rest of the code